### PR TITLE
Reconciliate Table Column Names

### PR DESCRIPTION
**Pull request by jmborr**
**Description of work.**
- [x] renamed column

**To test:**

Refs #30172

This does not require release notes because the affected algorithm is not yet in production

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by Harrietbrown** [comment]
I can;t access SCD166, could you copy the issue here?

**Comment by jmborr** [comment]
> I can;t access SCD166, could you copy the issue here?

Thanks for taking a look. This is the original problem, although some more problems cropped up:

> Algorithm CorelliPowderCalibrationCreate creates an adjustment table where the first column is named "Component". Algorithm
CorelliPowderCalibrationDatabase is expecting that column to be named "ComponentName".
To-do: In algorithm CorelliPowderCalibrationCreate, rename the column as "ComponentName". The remaining Other algorithms

File: Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h [hunk 1]
```diff
@@ -41,7 +41,7 @@ class MANTID_ALGORITHMS_DLL CorelliPowderCalibrationLoad
 
   /// Extra help info
   const std::vector<std::string> seeAlso() const override {
-    return {"CorelliPowderCalibrationGenerate "
+    return {"CorelliPowderCalibrationCreate "
```
Comment (peterfpeterson) [inline Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h:44]: Should be
```C
return {"CorelliPowderCalibrationGenerate", "CorelliPowderCalibrationDatabase", "CorelliPowderCalibrationApply"};
```
so the names can be resolved into links in the user documentation.

File: Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationDatabase.h [hunk 2]
```diff
@@ -126,8 +126,17 @@ class MANTID_ALGORITHMS_DLL CorelliPowderCalibrationDatabase
   const std::string category() const override {
     return "Diffraction\\Calibration";
   };
+
+  /// Extra help info
+  const std::vector<std::string> seeAlso() const override {
```
Comment (peterfpeterson) [inline Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationDatabase.h:131]: Should be
```C
    return {"CorelliPowderCalibrationCreate",  "CorelliPowderCalibrationLoad",
            "CorelliPowderCalibrationApply"};
```

File: Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h (hunk 1 above)
Comment (jmborr) [reply Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h:44]: I'm reading now `std::vector<std::string>` and realized I parked my brain in a far far away place :facepalm:

**Review by peterfpeterson — APPROVED** [review]
Lots of new comments and some additional tests.

**Comment by jmborr** [comment]
@rosswhitfield @peterfpeterson

**Pull request merged**